homeassistant/components/lektrico/strings.json

```diff
@@ -22,6 +22,17 @@
     }
   },
   "entity": {
+    "button": {
+      "charge_start": {
+        "name": "Charge start"
+      },
+      "charge_stop": {
+        "name": "Charge stop"
+      },
+      "reboot": {
+        "name": "Reboot"
```

Comment:
```suggestion
```

Reply:
I removed the reboot button from strings.json, thank you.